migrations/educandu-2022-01-21-01-add-render-media-to-markdown-and-annotation.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line camelcase
+export default class Educandu_2022_01_21_01_add_render_media_to_markdown_and_annotation {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async up() {
+    const updateFunc = section => {
+      section.content.renderMedia = false;
+    };
+    await this.updateSectionsInCollection('documentRevisions', ['markdown', 'annotation'], updateFunc);
+    await this.updateSectionsInCollection('documents', ['markdown', 'annotation'], updateFunc);
+  }
+
+  async down() {
+    const updateFunc = section => {
+      delete section.content.renderMedia;
+    };
+
+    await this.updateSectionsInCollection('documentRevisions', ['markdown', 'annotation'], updateFunc);
+    await this.updateSectionsInCollection('documents', ['markdown', 'annotation'], updateFunc);
+  }
+
+  async updateSectionsInCollection(collectionName, sectionTypes, updateFunc) {
+    const filter = { $or: sectionTypes.map(st => ({ 'sections.type': st })) };
+    await this.updateAll(collectionName, filter, doc => {
+      const hasUpdates = doc.sections.reduce((accu, section) => accu || this.tryUpdateSection(section, sectionTypes, updateFunc), false);
+      // eslint-disable-next-line no-console
+      console.log(`Collection: ${collectionName}, ID: ${doc._id} -> ${hasUpdates ? 'updated' : 'not modified'}`);
```

Comment:
You could maybe put this log after line 55 and only log the relevant information, I think "not modified" might just pollute the logs.

Reply:
Yes, changed